ops/README.md

```diff
@@ -89,15 +89,15 @@ user: admin
 password: mantle
 ```
 
-## Data persistance
+## Data persistence
 
 Grafana data is not currently saved. Any modifications or additions will be lost on container restart.
 
 InfluxDB is persisting data to a Docker volume.
 
 **Stopping the project removing the containers will not clear this volume**
 
-To remove the influxdb and grafana data, run a commands like
+To remove the influxdb and grafana data, run a command like
```

Comment:
`run commands like` may be better

Reply:
@Tri-stone I can change this but it's "a" command so normally it should be singular